apps/predbat/predbat.py

```diff
@@ -177,112 +202,199 @@
+        "default": 1.0,
     },
+    {"name": "metric_cloud_enable", "friendly_name": "Simulate clouds (beta)", "type": "switch", "default": False},
     {
-        "name": "metric_octopus_saving_rate",
-        "friendly_name": "Octopus Saving session assumed rate",
+        "name": "pv_scaling",
+        "friendly_name": "PV Scaling",
         "type": "input_number",
-        "min": 1,
-        "max": 500,
-        "step": 5,
+        "min": 0,
+        "max": 2.0,
+        "step": 0.01,
+        "unit": "multiple",
+        "icon": "mdi:multiplication",
+        "default": 1.0,
+    },
+    {
+        "name": "pv_metric10_weight",
+        "friendly_name": "Metric 10 Weight",
+        "type": "input_number",
+        "min": 0,
+        "max": 1.0,
+        "step": 0.01,
         "unit": "fraction",
-        "icon": "mdi:currency-usd",
+        "icon": "mdi:percent",
+        "default": 0.15,
     },
-    {"name": "metric_cloud_enable", "friendly_name": "Simulate clouds (beta)", "type": "switch"},
     {
-        "name": "set_window_minutes",
-        "friendly_name": "Set Window Minutes",
+        "name": "load_scaling",
+        "friendly_name": "Load Scaling",
         "type": "input_number",
-        "min": 5,
-        "max": 720,
-        "step": 5,
-        "unit": "minutes",
-        "icon": "mdi:timer-settings-outline",
+        "min": 0,
+        "max": 2.0,
+        "step": 0.01,
+        "unit": "multiple",
+        "icon": "mdi:multiplication",
+        "default": 1.0,
     },
     {
-        "name": "set_soc_minutes",
-        "friendly_name": "Set SOC Minutes",
+        "name": "battery_rate_max_scaling",
+        "friendly_name": "Battery rate max scaling",
         "type": "input_number",
-        "min": 5,
-        "max": 720,
-        "step": 5,
-        "unit": "minutes",
-        "icon": "mdi:timer-settings-outline",
+        "min": 0,
+        "max": 1.0,
+        "step": 0.01,
+        "unit": "multiple",
+        "icon": "mdi:multiplication",
+        "default": 1.0,
     },
-    {"name": "set_reserve_min", "friendly_name": "Set Reserve Min", "type": "input_number", "min": 4, "max": 100, "step": 1, "unit": "%", "icon": "mdi:percent"},
     {
         "name": "rate_low_threshold",
         "friendly_name": "Rate Low Threshold",
         "type": "input_number",
-        "min": 0.00,
-        "max": 2.00,
+        "min": 0.0,
+        "max": 2.0,
         "step": 0.05,
         "unit": "multiple",
         "icon": "mdi:multiplication",
+        "default": 0.8,
     },
     {
         "name": "rate_high_threshold",
         "friendly_name": "Rate High Threshold",
         "type": "input_number",
-        "min": 0.00,
-        "max": 2.00,
+        "min": 0.0,
+        "max": 2.0,
         "step": 0.05,
         "unit": "multiple",
         "icon": "mdi:multiplication",
+        "default": 0.0,
     },
-    {"name": "car_charging_hold", "friendly_name": "Car charging hold", "type": "switch"},
```

Comment:
This diff seems odd?

Reply:
I changed the order of the definitions in CONFIG_ITEMS so I could QC them against the various attributes that are initialised by them in fetch_config (see next comment). So the diffs look odd but it's just because of the change in order